opencti-platform/opencti-front/src/private/components/common/containers/ContainerHeader.jsx

```diff
@@ -763,6 +765,8 @@ const ContainerHeader = (props) => {
             name={t('Report representation')}
             pixelRatio={currentMode === 'graph' ? 1 : 2}
             adjust={adjust}
+            reportId={type === 'report' ? container.id : null}
```

Comment:
I'm wondering if we don't already have container.entity_type to check if it's a report, so we won't need "type"

Reply:
We don't, `container` has a few properties (its `id`, `name`, the objects that are displayed in the graph and some metadata). 

That's why, I had to enforce the type when used in a report knowledge graph context. Surely there is something better to do though, but it was not obvious to me. Open to suggestions